cooldowns_rogue.lua

```diff
@@ -293,4 +293,41 @@ LCT_SpellData[277925] = {
 	specID = { SPEC_ROGUE_SUB },
 	talent = true,
 	cooldown = 60
-}
\ No newline at end of file
+}
+
+-- Covenant Abilities
+-- Echoing Reprimand
+LCT_SpellData[323547] = {
+	class = "ROGUE",
+	covenant = "KYRIAN",
+	offensive = true,
+	cooldown = 45
+}
+
+-- Flagellation
+LCT_SpellData[323654] = {
+	class = "ROGUE",
+	covenant = "VENTHYR",
+	offensive = true,
+	duration = 20,
+	cooldown = 90
+}
+
+-- Serrated Bone Spike
+LCT_SpellData[328547] = {
+	class = "ROGUE",
+	covenant = "NECROLORD",
+	offensive = true,
+	duration = 10,			-- this figure is a wild stab in the dark, needs testing in PvP

```

Comment:
can just leave it out honestly, if I remember correctly it dispels itself when full HP